pkg/wmiext/instance.go

```diff
@@ -497,11 +497,7 @@ func (i *Instance) NextAsVariant() (bool, string, *ole.VARIANT, CIMTYPE_ENUMERAT
 		return true, "", nil, cimType, flavor, nil
 	}
 
-	defer func() {
-		if err := ole.SysFreeString((*int16)(unsafe.Pointer(strName))); err != nil {
-			logrus.Error(err)
```

Comment:
Should we logrus.Debug it instead?

Reply:
the error message is actually not helpful.   @n1hility is there a way we can conditionalize this SySFreeString call?

Reply:
The underlying win32 function call is defined with a void return type, so the vendored go-ole method here is mis-specified, so safe to ignore: 

https://learn.microsoft.com/en-us/windows/win32/api/oleauto/nf-oleauto-sysfreestring